adagrams/game.py

```diff
@@ -1,13 +1,262 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+}
+
+def letter_pool_as_list():
+    """Creates a list of the available letters.
+
+    Args:
+        none
+    
+    Returns:
+        list: letter_pool_list (list of strings). Each string is a single capital letter. The number of each capital letter depends on fluency as defined in LETTER_POOL.
+    """
+    letter_pool_list = []
+    for letter, frequency in LETTER_POOL.items():
+        for i in range(0, frequency):
+            letter_pool_list.append(letter)
+
+    return letter_pool_list
+
+STARTING_LETTER_LIST = letter_pool_as_list()
+MAX_INDEX_LETTER_LIST = len(STARTING_LETTER_LIST) - 1
+
 def draw_letters():
-    pass
+    """Simulates drawing 10 tiles from a pile. Accounts for letter distribution.
+
+    Args:
+        None
+
+    Returns:
+        list: tiles (list of strings)
+    """
+    tiles = []
+    while len(tiles) < 10:
+        random_index = randint(0, MAX_INDEX_LETTER_LIST)
+        letter = STARTING_LETTER_LIST[random_index]
+
+        if tiles.count(letter) < LETTER_POOL[letter]:
+            tiles.append(letter)
+        else:
+            continue
+
+    return tiles
 
 def uses_available_letters(word, letter_bank):
-    pass
+    """Checks if a player-submitted word only uses letters from the player's letter bank.
+
+    Args:
+        word (str): The word submitted by the player.
+        letter_bank (list): The list of letters the player has from drawing 10 tiles.
+
+    Returns:
+        bool: True if all letters in the submitted word are from letter pool. False if not. Will also return False if a letter's frequency in the word is greater than its frequency in the letter_bank.
+    """
+    capitalize_word = word.upper()
+    for letter in capitalize_word:
+        if letter not in letter_bank:
+            return False
+        else:
+            if capitalize_word.count(letter) > letter_bank.count(letter):
+                return False
+    return True
```

Comment:
Consider adding a few blank lines within your function to separate logically distinct sections of the code (e.g., setup, processing, return).  According to [PEP 8](https://peps.python.org/pep-0008/#blank-lines), blank lines can improve readability by visually grouping related operations. 

```suggestion
                return False
                
    return True
```